src/app/search/facet-search/facet-search.pipe.ts

```diff
@@ -14,7 +14,15 @@ import { Pipe, PipeTransform } from '@angular/core';
  */
 export class GetFacetSearchResultsPipe implements PipeTransform {
   transform(items: Array<any>, searchText: string, facet: string): Array<any> {
-    if (!items || !searchText || facet !== 'author') {
+    if (
+      !items ||
+      !searchText ||
+      (facet !== 'author' &&
+        facet !== 'organization' &&
+        facet !== 'labels.value.keyword' &&
+        facet !== 'namespace' &&
+        facet !== 'categories.name.keyword')
```

Comment:
Maybe we could encapsulate this list of ES fields in a single function/data structure which we could use throughout.

Reply:
Adding this to https://github.com/dockstore/dockstore/issues/5072